### PR TITLE
Add local GeoServer instance

### PR DESCRIPTION
**Pull request by lucernae**
Add local GeoServer setup

### CODE REVIEW

File: deployment/docker-compose.yml [hunk 1]
```diff
@@ -100,6 +100,7 @@ services:
       - osm-db:osm-db
       - rabbitmq:rabbitmq
       - worker:worker
+      - geoserver:geoserver
```
Comment (meomancer) [inline deployment/docker-compose.yml:103]: hi @lucernae 
can you remove this?

File: deployment/docker-compose.yml [hunk 2]
```diff
@@ -206,6 +207,7 @@ services:
       - smtp:smtp
       - db:db
       - osm-db:osm-db
+      - geoserver:geoserver
```
Comment (meomancer) [inline deployment/docker-compose.yml:210]: and also this?
can you remove this?

**Comment by meomancer** [comment]
hi @lucernae 
Can you create geoserver to be spin up on it's own make script?

**Comment by lucernae** [comment]
Ok Noted @meomancer

**Pull request closed without merging**